types/index.d.ts

```diff
@@ -102,8 +102,8 @@ export class Canvas {
 	toBuffer(mimeType: 'application/pdf', config?: PdfConfig): Buffer
 
 	/**
-	 * Returns the raw ARGB data without encoding in native-endian byte order,
-	 * top-to-bottom.
+	 * Returns the raw ARGB data without encoding, top-to-bottom. On big-endian
+	 * platforms, the data will instead be BGRA.
```

Comment:
This comment is a bit confusing.
Shouldn't `LE` be `BGRA`, and `BE` be `ARGB`?
As I read the text, it gives me the impression that it's the other way around

Reply:
Good catch! thank you